README.md

```diff
@@ -1,36 +1,28 @@
 [![Promote charm](https://github.com/canonical/gateway-api-integrator-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/gateway-api-integrator-operator/actions/workflows/promote_charm.yaml)
 [![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
 
-A Juju charm that deploys and manages a [Gateway API integator](https://gateway-api.sigs.k8s.io/) on Kubernetes.
-Gateway API is an official Kubernetes project focused on L4 and L7 routing in Kubernetes. This
-project represents the next generation of Kubernetes Ingress, Load Balancing,
-and Service Mesh APIs. From the outset, it has been designed to be generic,
-expressive, and role-oriented.
+A Juju charm that deploys and manages a [Gateway API integator](https://gateway-api.sigs.k8s.io/) on Kubernetes. Gateway API is an official Kubernetes project focused on L4 and L7 routing in Kubernetes. This project represents the next generation of Kubernetes Ingress, Load Balancing, and Service Mesh APIs. From the outset, it has been designed to be generic, expressive, and role-oriented. This charm simplifies initial deployment and "day N" operations of Gateway API Integrator. It allows for deployment on many different Kubernetes platforms, including [Canonical K8s](https://ubuntu.com/kubernetes). 
 
-This charm simplifies initial deployment and "day N" operations of Gateway API
-Integrator. It allows for deployment on many different Kubernetes platforms,
-including [Canonical K8s](https://ubuntu.com/kubernetes).
+As such, the charm makes it easy for those looking to take control of their own Gateway API deployment while keeping operations simple, and gives them the freedom to deploy on the Kubernetes platform of their choice.
 
-As such, the charm makes it easy for those looking to take control of their
-own Gateway API deployment while keeping operations simple, and gives them the
-freedom to deploy on the Kubernetes platform of their choice.
+For DevOps or SRE teams this charm will make operating a Gateway API simple and straightforward through Juju's clean interface. It will allow easy deployment into multiple environments for testing of changes.
 
-For DevOps or SRE teams this charm will make operating a Gateway API simple and
-straightforward through Juju's clean interface. It will allow easy deployment
-into multiple environments for testing of changes.
+For information about how to deploy, integrate, and manage this charm, see the Official [gateway-api-integrator charm Documentation](https://charmhub.io/gateway-api-integrator/docs).
 
-## Project and community
+## Get started
+To begin, refer to the [Getting Started](https://charmhub.io/gateway-api-integrator/docs/tutorial-getting-started) tutorial for step-by-step instructions.
+
+## Learn more
+* [Read more](https://charmhub.io/gateway-api-integrator/docs)
 
```

Comment:
Are there developer docs we can point to here?

I think it would be good to also list the official webpage: https://gateway-api.sigs.k8s.io

Reply:
We don't have any developer docs at the moment. I've added the official webpage link though.